pkg/dhclient/dhcp4.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+// DefaultScheme for boot file if there are none in the lease
+var DefaultScheme = "tftp"
```

Comment:
Changing the variable globally is a bit iffy. Is it reasonably possible to pass it as a parameter?

Reply:
That would mean changing the Lease.Boot() interface which and unnecessarily implementing it for v6.
Just for my own education, why would the global be frowned on here? Did you want setters/getters?